Entity/Notification.php

```diff
@@ -45,11 +45,14 @@
      */
     protected $createdAt;
 
-    public function __construct()
+    public function __construct(string $title, string $message, array $recipients)
     {
         $this->id              = Uuid::uuid4();
         $this->recipientsItems = new ArrayCollection();
         $this->createdAt       = new \DateTime();
+        $this->setTitle($title);
```

Comment:
`$this->title = $title`?

Reply:
It was like this, but I changed it.  I was thinking if I add some logic to setter...